.gitmodules

```diff
@@ -42,3 +42,6 @@
 	path = third-party/ffmpeg-macos-aarch64
 	url = https://github.com/LizardByte/build-deps
 	branch = ffmpeg-macos-aarch64
+[submodule "third-party/nanors"]
+	path = third-party/nanors
+	url = https://github.com/sleepybishop/nanors.git
```

Comment:
I know this repo only has one branch, but would you mind adding specifying the branch anyway?